Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the WMCO_VERSION as arg of the bundle target (e.g make bundle WMCO_VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export WMCO_VERSION=0.0.2)
-WMCO_VERSION ?= 9.0.0
+WMCO_VERSION ?= 10.15.0
```

Comment:
Shouldn't this update be done using the hack script? Unless the script cannot handle the bump from 9.0.0 to 10.15.0.

Reply:
yes the script needs to be updated now
cc: @wgahnagl